larray/core/axis.py

```diff
@@ -274,6 +274,64 @@ def extend(self, labels):
         labels = self._length + other._length if self.iswildcard else np.append(self.labels, other.labels)
         return Axis(labels, self.name)
 
+    def split(self, sep='_', names=None, regex=None, return_labels=False):
+        """Split axis and returns a list of Axis.
+
+        Parameters
+        ----------
+        sep : str, optional
+            Delimiter to use for splitting. Defaults to '_'.
+
+            When `regex` is provided, the delimiter is only used on `names` if given as one string or on axis name if
+            `names` is None.
+        names : str or list of str, optional
+            Names of resulting axes. Defaults to None.
+        regex : str, optional
+            Use regex instead of delimiter to split labels. Defaults to None.
+        labels : bool, optional
+            Whether or not split labels must be returned (as a tuple of tuples). These labels are suitable for indexing
+            via array.points[labels]. Defaults to False.
```

Comment:
Do you want to open a new issue to add that option to LArray.split_axes?

Reply:
LArray.split_axes? I don't see a use case. In AxisCollection.split_axes, that **might** make the code of LArray.split_axes nicer/smaller, but I doubt the array.points[] = value with multiple labels (for the several axes) will work. I haven't thought much about that though.